R/lvl0_mergeDat.R

```diff
@@ -129,6 +130,13 @@ lvl0_mergeDat = function(UCD_output, EU_data, PSI_costs, altCosts, CHN_trucks, G
   purchCost[, tot_purchasecost := valUCD]
   purchCost[, valUCD := NULL]
 
+  ## merge with GDP_MER to rescale the capital costs in terms with the aim of representing 2nd hand market
+  GDPcoeff = copy(GDP_MER)
+  GDPcoeff[, factor := ifelse(weight<= 30000 & weight >= 4000, (1-0.3)/(30000-4000)*(weight-4000)+0.3, 1)]
```

Comment:
would be good to define the paremeters on top of these lines, e.g.,
```
min_gdp <- 4000
max_gdp <- 30000
lower_bound <- 0.3
```

Reply:
thanks for pointing it out - addressed in the new commit